<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>1041</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1041</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_file">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Выбор файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_f">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter';
	border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="GLView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>768</width>
      <height>1024</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertexes">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter';
	border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_facets">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter';
	border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вершины</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ребра</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_creator">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Отобразить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Color">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>610</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QFrame" name="move">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_dx">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>30</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Перемещение по оси:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_dy">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>60</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_dz">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>90</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="scale">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>270</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_scale">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Масштаб</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Projection">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Проекция</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ortho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Параллельная</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_frustum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Центральная</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="lines">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_width">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Линии</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>толщина</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Color_line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_linetype_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>сплошные</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_linetype_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>пунктирные</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Points">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>790</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_pWidth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Вершины</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>размер</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Color_point">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_point_none">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>отсутствуют</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_point_circle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>круг</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_point_square">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>квадрат</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_screenshot">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>940</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>PNG</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>PNG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPEG</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_screenshot">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>940</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Gif">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>990</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Record to Gif</string>
    </property>
   </widget>
   <widget class="QFrame" name="rotate">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_rotate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(170, 170, 255);
	color: rgb(0, 0, 0);
	border: 1px solid white;
}
QPushButton:pressed {
	color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(203, 36, 168, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
     </property>
     <property name="text">
      <string>Поворот на угол по оси:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_z">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLView</class>
   <extends>QWidget</extends>
   <header>glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
